src/Apps/Artist/Components/NavigationTabs.tsx

```diff
@@ -62,7 +63,7 @@ export class NavigationTabs extends React.Component<Props> {
 
     return (
       <>
-        {this.renderTab("Overview", route(""), {
+        {this.renderTab("Overview", route(user ? "/overview" : ""), {
```

Comment:
Ah-- will this break now that you made the other change?

Reply:
Nope, this will continue to function fine. Both `/` and `/overview` will lead to the overview route.

Reply:
The code to support that is here: https://github.com/artsy/reaction/pull/3338/files#diff-53ea779b20e5bd37af12c435a46415edR145

Reply:
Ahhh I was misunderstanding what this option was even for 😅 . Assumed it was about matching/highlighting.

Reply:
Actually, @sweir27 and I talked about this a bit offline. After doing some testing I saw the above change actually _does_ break tab highlighting which I think was the original reason for adding it. Gonna re-add.